plugins/modules/ilo_redfish_config.py

```diff
@@ -148,11 +482,22 @@ def main():
              "token": module.params['auth_token']}
 
     timeout = module.params['timeout']
+    if module.params['timeout'] is None:
+        timeout = 10
+        module.deprecate(
+            'The default value {0} for parameter param1 is being deprecated and it will be replaced by {1}'.format(
+                10, 60
+            ),
+            version='7.0.0',
```

Comment:
The standard deprecation cycle is roughtly two major versions (i.e. roughly a year).
```suggestion
            version='8.0.0',
```